packages/eslint-plugin/src/rules/no-var-requires.ts

```diff
@@ -13,10 +13,10 @@ type MessageIds = 'noVarReqs';
 export default createRule<Options, MessageIds>({
   name: 'no-var-requires',
   meta: {
+    deprecated: true,
```

Comment:
do you mind adding missing meta field, `replacedBy`
https://eslint.org/docs/latest/extend/custom-rules#rule-structure